widget_driver/pubspec.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.7.0
-  widget_driver_annotation: ^0.0.1
+  widget_driver_annotation: 0.0.2
```

Comment:
We need to have this hard coded for now.. Since otherwise the widget driver is grabbing the new annotations.. which have breaking changes.. In my next PR I am fixing this